src/main/scala/ru/tinkoff/gatling/amqp/protocol/package.scala

```diff
@@ -7,25 +7,25 @@ import ru.tinkoff.gatling.amqp.request.AmqpProtocolMessage
 
 package object protocol {
   trait AmqpMessageMatcher {
-    def prepareRequest(msg: AmqpProtocolMessage): Unit
+    def prepareRequest(msg: AmqpProtocolMessage): AmqpProtocolMessage
     def requestMatchId(msg: AmqpProtocolMessage): String
     def responseMatchId(msg: AmqpProtocolMessage): String
   }
 
   object MessageIdMessageMatcher extends AmqpMessageMatcher {
-    override def prepareRequest(msg: AmqpProtocolMessage): Unit    = {}
+    override def prepareRequest(msg: AmqpProtocolMessage): AmqpProtocolMessage    = msg
```

Comment:
May you move this in trait AmqpMessageMatcher and override only in CorrelationIdMessageMatcher

Reply:
Sure.  Done.